src/serde.rs

```diff
@@ -276,13 +276,23 @@ pub struct AccountData {
     /// balance
     #[serde(deserialize_with = "de_uint_hex", serialize_with = "se_uint_hex")]
     pub balance: BigUint,
-    /// codeHash
+    /// keccakCodeHash
```

Comment:
still I prefer naming it as `codeHash` here so we can kept compatible on code with zkevm-circuit

Reply:
we were actually planning to rename it to `keccakCodeHash` in https://github.com/scroll-tech/zkevm-circuits/pull/309, but I think we can undo it there too.